docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -374,7 +374,7 @@ All named curves are instances of :class:`EllipticCurve`.
     SECG curve ``secp256r1``. Also called NIST P-256.
 
 
-.. class:: SECT224R1
+.. class:: SECP224R1
```

Comment:
:+1: This indeed is [missing a source link](https://cryptography.io/en/stable/hazmat/primitives/asymmetric/ec/#cryptography.hazmat.primitives.asymmetric.ec.SECT224R1).